packages/extension-ui/src/Popup/CreateAccount/SafetyFirst.tsx

```diff
@@ -22,22 +22,32 @@ function SafetyFirst({ className, onNextStep }: Props): React.ReactElement<Props
 
   const goTo = useCallback((path: string) => () => onAction(path), [onAction]);
 
+  const CustomFooter = styled(HelperFooter)`
+   .wrapper-row {
+    display: flex;
+    flex-direction: row;
+    gap: 8px;
+    width: 100%;
+   }`;
```

Comment:
Not sure how this works but maybe instead of modifying `HelperFooter` you could  create styled `div` called `WrapperRow` and put it under HelperFooter? All elements will still be children of HelperFooter so the styling for link etc should be applied I think 🤔

Reply:
Also I would put it out of the component to avoid re-renders.

Reply:
`styled(HelperFooter)` creates a new component (= class) under which `.wrapper-row` is nested in the hierarchy - it no longer is going to be the `HelperFooter`. If you want to achieve nesting, you'd have to follow [this syntax](https://styled-components.com/docs/advanced#referring-to-other-components):
```ts
const Ancestor = styled.xxx`
  ${Descendant} {
    ...
  }
`
```

But I agree with @plakak - it's generally better to have a separate styled component. One of Styled Components advantages is that it doesn't require you to nest any classes, because nesting is troublesome to maintain and to reason about.